src/main/java/com/example/javawebcourse/config/FeatureToggleProperties.java

```diff
@@ -0,0 +1,22 @@
+package com.example.javawebcourse.config;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Map;
+
+@Data
+@NoArgsConstructor
+@Configuration
```

Comment:
@ConfigurationProperties already marks class as bean, no need for @Configuration annotation